Src/FluentAssertions/FluentAssertions.csproj

```diff
@@ -14,7 +14,7 @@
     <Authors>Dennis Doomen;Jonas Nyrup</Authors>
     <PackageDescription>
       A very extensive set of extension methods that allow you to more naturally specify the expected outcome of a TDD or
-      BDD-style unit tests. Targets .NET Framework 4.5 and 4.7, as well as .NET Core 2.0, .NET Standard 1.3, 1.6 and 2.0.
+      BDD-style unit tests. Targets .NET Framework 4.5 and 4.7, as well as .NET Core 2.0, .NET Core 3.0, .NET Standard 1.3, 1.6 and 2.0.
```

Comment:
We're no longer targeting net core 3.0

Reply:
But we do support it officially. So I would keep it in.

Reply:
Maybe the PR can also update the `docs` folder.

Reply:
Done.
Please check.